apps/vr-tests/src/utilities/index.ts

```diff
@@ -8,7 +8,7 @@ export interface IStoryConfig {
   rtl?: boolean;
 }
 
-declare module '@storybook/addons/dist/types' {
+declare module '@storybook/addons/dist/ts3.9/types' {
```

Comment:
q: so they(storybook) moved legacy API typed parts under down-leveled declaration types ?

Reply:
Yeah - addressed by this [PR](https://github.com/storybookjs/storybook/pull/13013)